deploy-board/deploy_board/webapp/group_view.py

```diff
@@ -24,10 +24,9 @@
 import logging
 import traceback
 
-from helpers import environs_helper, clusters_helper
-from helpers import groups_helper, baseimages_helper
-from helpers import specs_helper, autoscaling_groups_helper
-from helpers import autoscaling_metrics_helper
+from helpers import environs_helper, clusters_helper, groups_helper,\
```

Comment:
nit: prefer black style or just leave as is: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#parentheses

Reply:
we should probably choose a standard as a project 😃